src/functions/asyncCredential/asyncCredentialHandler.test.ts

```diff
@@ -647,6 +647,36 @@ describe("Async Credential", () => {
     });
   });
 
+  // describe("Request body validation - using Client Credentials", () => {
```

Comment:
Please may we not comment this test out - it's really important it constantly fails - both in the Github action and locally :)

Reply:
The pre-commit hook stops us from pushing with failing tests. Happy to comment out that part of the hook if needed

Reply:
right you are - I assumed there was a reason!